packages/varnish/src/components/header/AI2Banner.tsx

```diff
@@ -17,20 +21,27 @@ const BannerContent = styled(Content)`
   padding-bottom: 0;
 `;
 
-const DarkBanner = styled.div`
-  background: ${({ theme }) => theme.paletteExtended.background.dark};
+const StyledBanner = styled.div<{ blackAndWhite?: boolean }>`
```

Comment:
i am wondering if we need to alter the footer and content sections to also be monochrome
in which case, we may move this to context

Reply:
could we see a world where users want just an individual piece (such as the banner) to be black and white while everything else stays the same? though this is more tedious because we have to do it in each of the 3 components you just mentioned (banner, footer, content), it provides more granular control. 
we could have a 'theme' setting in the context that defaults all of these to true if monochrome / B&W is enabled...?

Reply:
lets just keep it in the back of our mind
do we need a black footer for olmo?

Reply:
checking with Matt on that...

Reply:
i'll put out another footer PR depending on his feedback per what the footer styling should look like; pushed iterations to this PR based on other feedback though!